### PR TITLE
ossia-score: 3.2.3-3 -> 3.2.4

### PR DESCRIPTION
**Pull request by jcelerier**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by jcelerier** [comment]
Updated to 3.2.4 and also switched the build to clang to support aarch64 (and also better DSP performance)

**Comment by jcelerier** [comment]
@SuperSandro2000 @Aleksanaa @magnetophon

File: pkgs/by-name/os/ossia-score/package.nix [hunk 1]
```diff
@@ -42,15 +42,15 @@
 # TODO: figure out LLVM jit
 # assert lib.versionAtLeast llvm.version "15";
 
-stdenv.mkDerivation (finalAttrs: {
```
Comment (minijackson) [inline pkgs/by-name/os/ossia-score/package.nix:45]: you can remove the `stdenv` argument at the top

File: pkgs/by-name/os/ossia-score/package.nix [hunk 2]
```diff
@@ -42,15 +42,15 @@
...
   pname = "ossia-score";
-  version = "3.2.3-3";
+  version = "3.2.4";
 
   src = fetchFromGitHub {
     owner = "ossia";
     repo = "score";
-    rev = "v3.2.3-3";
-    hash = "sha256-xRqsMKwuejbl+5ljYMFhQv/j1MfnFH5MGIn9rCQG/ro=";
+    rev = "v3.2.4";
```
Comment (minijackson) [inline pkgs/by-name/os/ossia-score/package.nix:52]: It might be better to just do this:

```suggestion
    rev = "v${finalAttrs.version}";
```

**Review by minijackson — COMMENTED** [review]
Tested and it works. Thanks! Some small nitpicks due to the change.

---

Result of `nixpkgs-review pr 325819` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>ossia-score</li>
  </ul>
</details>

File: pkgs/by-name/os/ossia-score/package.nix (hunk 1 above)
Comment (jcelerier) [reply pkgs/by-name/os/ossia-score/package.nix:45]: done

**Comment by minijackson** [comment]
@jcelerier I think you still need to squash all your commits in a single one

**Comment by jcelerier** [comment]
Oh I thought this would be done automatically. Doing that asap

**Review by magnetophon — APPROVED** [review]
Result of `nixpkgs-review pr 325819` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>ossia-score</li>
  </ul>
</details>

**Comment by jcelerier** [comment]
rebased and squashed

**Review by minijackson — APPROVED** [review]
Result of `nixpkgs-review pr 325819` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>ossia-score</li>
  </ul>
</details>

**Pull request merged**